Deploy VM as the sample app instead of a Sleep pod

kirankt opened the pull request:
Increase size of PVC to 5G to accommodate the VM example
Update README with instructions for VM centric deployment.

Commit:
Deploy VM as the sample app instead of a Sleep pod
Increase size of PVC to 5G to accommodate the VM example
Update README with instructions for VM centric deployment.

cschwede commented on tools/demo.sh:
```diff
@@ -26,8 +27,12 @@ $_kubectl -n ember-csi wait --timeout=300s --for=condition=Ready pod/external-ce
+
+# Install virtctl
+mkdir -p _out/cmd/virtctl
+curl -L -o _out/cmd/virtctl/virtctl https://github.com/kubevirt/kubevirt/releases/download/$VERSION/virtctl-$KUBEVIRT_VERSION-linux-amd64
```
$VERSION needs to be $KUBEVIRT_VERSION

kirankt replied:
The new commit fixes this.

cschwede commented on README.md:
````diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
+
+```
+# Creating a virtual machine
+$ kubectl apply -f https://raw.githubusercontent.com/kubevirt/demo/master/manifests/vm.yaml
````
This command only works for me if I replace kubectl with ./cluster/kubectl.sh  (same for all following commands).

kirankt replied:
Since we are creating an alias, we can use that instead. The new commit uses this.

cschwede commented on README.md:
```diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
+$ kubectl get vms -o yaml testvm
+
+# To start a VM you can use
+$ ./virtctl start testvm
```
Should be ./cluster/virtctl.sh - "make all" was executed in the top level dir, so we need the leading prefix dir here and for all other commands

kirankt replied:
again, we will use the alias 'v' here.

cschwede commented on README.md:
````diff
@@ -12,9 +12,35 @@ This deployment deploys an ephemeral Ceph container alongside Ember CSI and its
-$ make deploy
-$ make demo
+$ make all
+```
````
I had to execute "source tools/env.sh" here as well to make it work

cschwede reviewed (changes requested):
A few small changes were required to make this work. Please have a look at my comments.

The PVC is also not used in this example, I think we should change this too.

kirankt committed:
Update demo.sh with instructions to create CDI
Update deploy.sh with instructions to remove extra pods to make space for new ones
Update env.sh with new and revised alias
Fix typo in demo.sh $VERSION->$KUBEVIRT_VERSION
Update README with instructions to create PVC and VM

kirankt committed:
Update demo.sh with instructions to create CDI
Update deploy.sh with instructions to remove extra pods to make space for new ones
Update env.sh with new and revised alias
Fix typo in demo.sh $VERSION->$KUBEVIRT_VERSION
Update README with instructions to create PVC and VM
Add cirros-pvc and cirros-vm examples

kirankt commented:
I have a new commit which uses Ember-CSI provisioned PVC to create a cirros VM. Please give it a whirl and review. Thanks.

kirankt commented:
The examples/cirros* files were missing. Added them.

cschwede commented:
LGTM - tested and works as expected.
